blog.html

```diff
@@ -0,0 +1,167 @@
+                        interest of lorem ipsum and mauris neque quam blog. I want to share my world with you.</p>
+                </div>
+                <img class="blog-img" src="./images/jl_y_yo.jpeg" alt="My boyfriend and I">
+            </div>
+            <hr>
+        </div>
+    <!-- END Introduction Menu -->
+        <!-- Blog entry -->
+        <div class="   ">
+            <div class= "">
+                <h3><b>MT. RAINER</b></h3>
+                <h5>An entry, <span class= "">Dec, 2019</span></h5>
+            </div>
+
+            <div class= "">
+                <p>Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem
+                    euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed
+                    ultricies mi non congue ullam corper. Praesent tincidunt sed
+                    tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida
+                    diam non fringilla.</p>
+                </div>
+                <img class="blog-img" src="./images/Mt_Rainer_by_AO.jpeg" alt="Mount Rainer">
+            </div>
+        <hr>
+
+<!-- Blog entry --> 
+    <div class="">
+        <div class= "">
+            <h3><b>THE PUGET SOUND</b></h3>
+            <h5>An entry, <span class= "">MAY, 2021</span></h5>
+        </div>
+        
+        <div class= "">
+            <p>Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem
+                euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed
+                ultricies mi non congue ullam corper. Praesent tincidunt sed
+                tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida
+                diam non fringilla.</p>
+                
+            </div>
+            <img class="blog-img" src="./images/a_favorite_color_AO.jpeg" alt="Vashon Island">
+    </div>
+    <hr>
+    <!-- Blog entry -->
+    <div class="   ">
+        <div class= "">
+            <h3><b>FEET FIRST</b></h3>
+            <h5>Title description, <span class= "">Dec, 2020</span></h5>
+        </div>
+
+        <div class= "">
+            <p>Mauris neque quam, fermentum ut nisl vitae, convallis maximus nisl. Sed mattis nunc id lorem
+                euismod placerat. Vivamus porttitor magna enim, ac accumsan tortor cursus at. Phasellus sed
+                ultricies mi non congue ullam corper. Praesent tincidunt sed
+                tellus ut rutrum. Sed vitae justo condimentum, porta lectus vitae, ultricies congue gravida
+                diam non fringilla.</p>
+            </div>
+            <img class="blog-img" id="feet" src="./images/oceanic_feet.jpeg" alt="Image of feet feeling the sand and ocean water">
+        </div>
+    <hr>
+
+<!-- Blog entry -->
+    <!-- Blog entry -->
+    <div class="   ">
+        <div class= "">
```

Comment:
I noticed there are multiple classes with empty stringA div does not need to have a class. In general, we prefer divs for styling if they have classnames, and section if it's not for styling as it's considered semantic html.